### PR TITLE
Refactor to new version of the core

### PR DESCRIPTION
**Pull request by MahdiyarGHD**
### Checklist:

- [ ] It refers to an [Issue](https://github.com/EasyMicroservices/FileManager/issues).
- [ ] It fixes a bug.
- [ ] It's a new feature (non-breaking change which adds functionality)
- [ ] It's a breaking (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.
- [ ] Related language label is selected.

### CODE REVIEW

File: src/CSharp/EasyMicroservices.ProfilesMicroservice.Database/Database/Contexts/ProfileContext.cs [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+using EasyMicroservices.Cores.Relational.EntityFrameworkCore.Intrerfaces;
 
 namespace EasyMicroservices.ProfilesMicroservice.Database.Contexts
 {
     public class ProfileContext : RelationalCoreContext
     {
-        IDatabaseBuilder _builder;
-        public ProfileContext(IDatabaseBuilder builder)
+        IEntityFrameworkCoreDatabaseBuilder _builder;
+        public ProfileContext(IEntityFrameworkCoreDatabaseBuilder builder)
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.ProfilesMicroservice.Database/Database/Contexts/ProfileContext.cs]: please call base(builder)

File: src/CSharp/EasyMicroservices.ProfilesMicroservice.WebApi/Program.cs [hunk 2]
```diff
@@ -1,168 +1,38 @@
...
-using EasyMicroservices.ProfilesMicroservice.Interfaces;
+using EasyMicroservices.Cores.AspEntityFrameworkCoreApi;
+using EasyMicroservices.Cores.Relational.EntityFrameworkCore.Intrerfaces;
 using EasyMicroservices.ProfilesMicroservice;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Common;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Responses;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Requests;
 
-namespace EasyMicroservices.ProfilesMicroservice.WebApi
+namespace EasyMicroservices.ContentsMicroservice.WebApi
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.ProfilesMicroservice.WebApi/Program.cs]: it's Profile microservice, don't change the namespace please

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by Mohsen-Yaghoubi — APPROVED** [review]

File: src/CSharp/EasyMicroservices.ProfilesMicroservice.WebApi/Program.cs (hunk 2 above)
Comment (MahdiyarGHD) [reply src/CSharp/EasyMicroservices.ProfilesMicroservice.WebApi/Program.cs]: was a mistake

**Review by Mohsen-Yaghoubi — APPROVED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]

**Review by mahdidelzendeh72 — APPROVED** [review]
thanks

**Pull request merged**